src/character.h

```diff
@@ -484,6 +484,7 @@ class Character : public Creature, public visitable<Character>
         std::vector<bionic> my_bionics;
 
     protected:
+        virtual void on_stat_change( const std::string &, int ) override {};
```

Comment:
Would be better if it was an enum rather than `string` here.

Each stat has to be handled in code meaning the string offers no advantages here, but using it means no warnings on unhandled enums, no static warnings on invalid arguments (actually the whole possibility to actually specify invalid arguments) and slightly lower performance.